### DISCUSSION TITLE
Announcement Kura 5.4: FEATURE FREEZE 🥶 🚀

### DISCUSSION THREAD

**Discussion by GregoryIvo** [eclipse-kura/kura#4943] [Announcements]
As we gear up for Kura Release 5.4, we are excited to announce the commencement of an official feature freeze.

In this phase, Kura Development Will focus on:
-  QA testing
-  bug fixing
-  general stabilization of the Kura code base.

Stay tuned for upcoming Milestone Releases and Release Candidates! 🚀

**Comment by mattdibi** [comment]
[5.4 is out!](https://github.com/eclipse/kura/discussions/5039)
